package_manifest.yaml

```diff
@@ -60,7 +60,7 @@ foreman_scl_packages:
     rubygem-excon: {}
     rubygem-extlib: {}
     rubygem-facter: {}
-    rubygem-faraday: {}
+    rubygem-faraday:
```

Comment:
I thought this was already done in develop

Reply:
It was, I just needed to rebase this.